MaskRCNN/model/mask_head.py

```diff
@@ -11,19 +11,22 @@
 from model.backbone import GroupNorm
 from config import config as cfg
 from utils.mixed_precision import mixed_precision_scope
-
+from performance import print_buildtime_shape, print_runtime_shape
 
 @under_name_scope()
 def maskrcnn_loss(mask_logits, fg_labels, fg_target_masks):
     """
+    Calculate the loss for the mask
+
     Args:
-        mask_logits: #fg x #category xhxw
-        fg_labels: #fg, in 1~#class, int64
-        fg_target_masks: #fgxhxw, float32
+        mask_logits: Num_fg_boxes x num_category x H x W, H=W=28
+        fg_labels: 1-D Num_fg_boxes
+        fg_target_masks: Num_fg_boxes x 28 x 28, float32
```

Comment:
It doesn't have to be 28, right? I think `h` and `w` are dependent on the size of the inputs, not hard-coded values